extension/test/unit/config/init/ensure-resources.spec.js

```diff
@@ -61,27 +61,26 @@ describe('Ensure resources', () => {
     )
   })
 
-  it('should skip creating resources when they already exist', async () => {
```

Comment:
Why did you remove this test?

Reply:
it will not skip anymore instead it will check the diff and create actions, I tried to use but those tests are mostly based on mocking the API calls and error-prone, I wrote an integration test instead, unit tests might check error handling logic.